### DISCUSSION TITLE
css variables

### DISCUSSION THREAD

**Discussion by kebikec** [facelessuser/ColorHelper#272] [Ideas]
I guess it doesn't support the use of variables?
I see color where variable is defined. 
but I don't see the color when I assign it to a class. 

It would be nice if it can be added in the future. 

:root {
   --blue1 : #2e54ee;
   --blue2 : rgb(158, 198, 254);
}

.sacmaBir {
  color: var(--blue1);
}

**Comment by facelessuser** [comment]
No, it does not.
